src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -115,11 +115,11 @@ public async Task<IActionResult> CreatePledge(long accountId, [FromBody]CreatePl
 
         [HttpGet]
         [Route("accounts/{accountId}/pledges/create/amount")]
-        public async Task<IActionResult> Amount(string accountId)
+        public async Task<IActionResult> Amount(long accountId)
         {
             try
             {
-                var queryResult = await _mediator.Send(new GetAmountQuery{EncodedAccountId = accountId });
+                var queryResult = await _mediator.Send(new GetAmountQuery{ AccountId = accountId });
```

Comment:
See previous comment - now would be a good time to check if the queryresult is `null` and, if it is, return a `NotFound` response from this action.

Reply:
Done as suggested.